docs/bindings.rst

```diff
@@ -51,3 +51,26 @@ as a GET parameter. For example for ``pulp_rpm`` only endpoints use a query like
 The schema can then be used as input to the openapi-generator-cli. The documentation on getting
 started with openapi-generator-cli is available on
 `openapi-generator.tech <https://openapi-generator.tech/#try>`_.
+
+
+Generating a client on dev environment
```

Comment:
Might change this to "Generating a client for a dev environment" to make it a little clearer